config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -113,7 +113,7 @@ presubmits:
         # Skipping snat tests probably GCE related? https://github.com/kubernetes/test-infra/issues/20321
         # Skipping Cloud Provider specific tests: LoadBalancer, ESIPP (Source IP Preservation)
         - >
-          --test_args=--ginkgo.skip=GCE|Disruptive|Serial|SNAT --ginkgo.label-filter='(sig-network || Conformance || Feature: containsAny {NetworkPolicy, NetworkPolicyEndPort} ) && !Feature: containsAny {Alpha, Beta, Example, Federation, IPv6DualStack, LoadBalancer, Networking-IPv6, PerformanceDNS, SCTPConnectivity, KubeProxyDaemonSetMigration}'
+          --test_args=--ginkgo.skip=GCE|Disruptive|Serial|SNAT --ginkgo.label-filter=(sig-network || Conformance || Feature: containsAny {NetworkPolicy, NetworkPolicyEndPort} ) && !Feature: containsAny {Alpha, Beta, Example, Federation, IPv6DualStack, LoadBalancer, Networking-IPv6, PerformanceDNS, SCTPConnectivity, KubeProxyDaemonSetMigration}
```

Comment:
@pohly you went through this before, can you help us with the review?

Reply:
The quoting of "--test-args" content is insane. It is totally non-obvious how that string gets passed on and who breaks it into individual parameters (`--ginkgo.skip`, `--ginkgo.label-filter`, etc.).

For node tests, I gave up in https://github.com/kubernetes/test-infra/pull/32776 and just used the one variant that seemed to work, which was single quotes in YAML and double quotes inside the string.

Reply:
You are using YAML `- >` quoting here, so you don't need the outer quotes. Perhaps `--ginkgo.label-filter="..."` works if single quotes didn't.

I doubt that leaving them out entirely works.